src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/QueryReporter.kt

```diff
@@ -0,0 +1,35 @@
+package no.skatteetaten.aurora.gobo.graphql
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Component
+import java.time.LocalDateTime
+import java.util.concurrent.ConcurrentHashMap
+
+data class QueryOperation(val korrelasjonsid: String, val name: String, val klientid: String?, val query: String, val started: LocalDateTime)
+
+private val logger = KotlinLogging.logger { }
+
+@Component
+class QueryReporter {
+    private val queries = ConcurrentHashMap<String, QueryOperation>()
```

Comment:
Nyssgjerrig, hvorfor bruke en hashMap og ikke cache ?

Reply:
Jeg testet med caffine først, men vi trenger ikke eviction. Caffine er helt super når man feks skal fjernes etter en viss tid, men i vårt tilfelle skal vi fylle opp map'et helt til det logges ut og tømmes helt.

Jeg så egentlig ingen fordeler lenger med å bruke caffine da, men mulig du har andre argumenter for at vi bør bruke det?